fe/fe-core/src/main/java/org/apache/doris/load/loadv2/LoadingTaskPlanner.java

```diff
@@ -127,7 +127,12 @@ public void plan(TUniqueId loadId, List<List<TBrokerFileStatus>> fileStatusesLis
 
         // 2. Olap table sink
         List<Long> partitionIds = getAllPartitionIds();
-        OlapTableSink olapTableSink = new OlapTableSink(table, tupleDesc, partitionIds);
+        OlapTableSink olapTableSink;
+        try {
+            olapTableSink = new OlapTableSink(table, tupleDesc, partitionIds);
+        } catch (IllegalStateException e) {
```

Comment:
This is not a good way to catch `IllegalStateException` here. Because it is a "Assertion" check in OlapTableSink.
We should check if `partitionIds` is empty before creating new `OlapTableSink`.

And by the way, there are severval place where the OlapTableSink's constructor is called, we should modify them too.

Reply:
ok，I will check if partitionIds is empty before creating new OlapTableSink. And modify all where OlapTableSink's constructor is called.